app/controllers/concerns/idv_session.rb

```diff
@@ -2,7 +2,6 @@ module IdvSession
   extend ActiveSupport::Concern
 
   def confirm_idv_session_started
-    return if current_user.decorate.pending_profile_requires_verification?
```

Comment:
To me, this degrades the user experience. Before, if a user who had chosen USPS verification signs out and back in, then clicks "I can't get mail at this address", they were redirected to `/verify/phone` where they could choose to verify by phone. Now, if you click "send me another letter", and then click "I can't get mail at this address", you have to enter your personal info all over again even though we already have it saved. This is most likely an oversight and we probably meant to replace "I can't get mail at this address" on that screen with "Reset and restart identity verification".

Have we talked to the design team about this change? I personally think "I can't get mail at this address" was clearer than "Reset and restart identity verification". Why is resetting my only option? Why can't I switch to phone verification since I've already completed the previous steps?

Reply:
Yeah, we've talked about this with design (cc: @donjo).

The reason for this change is that the phone step depends on the verified state that is built during the profile step. When you log out, the work from that step is deleted with the session and the only parts that can be re-created are the parts that are saved with the profile.

Additionally, the flow is built with verify by phone as the primary preferred path and verify by mail as a fallback. Pushing users who failed phone verification and are electing to do USPS verification back into the phone flow is likely to be frustrating. What a user who can't receive mail at a give address likely wants to do is reverify with an address where they can receive mail.